src/sentry/api/endpoints/organization_details.py

```diff
@@ -266,7 +265,12 @@ def validate_projectRateLimit(self, value):
     def validate_providerName(self, value):
         from sentry.auth import manager
 
-        if not manager.exists(value):
+        organization = self.context["organization"]
+        request = self.context["request"]
```

Comment:
We're not reusing these variables, so it feels pointless to name them here.

Reply:
Removed